pwiz_tools/Skyline/CommandLine.cs

```diff
@@ -1193,19 +1193,22 @@ private bool SetPeptideDigestSettings(CommandArgs commandArgs)
                         p = p.ChangeFilter(p.Filter.ChangePeptideUniqueness(commandArgs.PeptideDigestUniquenessConstraint.Value));
                     }
 
-                    if (commandArgs.BackgroundProteomeName != null)
-                    {
-                        var bgProteome = Settings.Default.BackgroundProteomeList.GetBackgroundProteomeSpec(commandArgs.BackgroundProteomeName);
-                        p = p.ChangeBackgroundProteome(new BackgroundProteome(bgProteome));
-                    }
-
                     if (commandArgs.BackgroundProteomePath != null)
                     {
-                        string name = Path.GetFileNameWithoutExtension(commandArgs.BackgroundProteomePath);
+                        if (!File.Exists(commandArgs.BackgroundProteomePath))
+                            throw new IOException(string.Format(
+                                Resources.CommandLine_FindBackgroundProteome_Warning__Could_not_find_the_background_proteome_file__0__,

```

Comment:
Why is this a Warning but throwing an IO Exception? Seems like it should be an error anyway.

Reply:
Ah I didn't even notice that. I was just trying to re-use the existing resource. Kind of argues for not including "Error:" or "Warning:" in the message text, so the same message might be either an error or a warning depending on context. Those cows are certainly already out of the barn though.